storage/storage_tests/create_country_info_getter.cpp

```diff
@@ -0,0 +1,33 @@
+#include "testing/testing.hpp"
```

Comment:
File names should mirror what's inside the files, not what a particular function does. Please, rename this file to something like "helpers.hpp".